components/batch-system/benches/batch-system.rs

```diff
@@ -48,9 +48,9 @@ fn bench_spawn_many(c: &mut Criterion) {
     system.shutdown();
 }
 
-/// Bench how it performs if two hot FSMs are shown up at the same time.
+/// Bench how it performs if two hot Fsms are shown up at the same time.
```

Comment:
Why change?

Reply:
Just to unify the old "FSM" and "fsm". I think Fsm is slightly more good-looking than FSM. But I can switch it back if you have a stronger preference.

Reply:
FSM itself is an abbreviation. `Fsm` is a name in code. `FSM` explains the meaning itself while `Fsm` does not and needs extra explanation on what it is.